contracts/BMTFile.sol

```diff
@@ -0,0 +1,119 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "./BMTChunk.sol";
+
+contract BMTFile is BMTChunk {
+  // max segment count
+  uint64 public constant MAX_SEGMENT_COUNT = 128;
+
+  // chunk bmt levels
+  uint64 public constant CHUNK_BMT_LEVELS = 7;
+
+  struct ChunkInclusionProof{
+    uint64 span;
+    bytes32[] sisterSegments;
+  }
+
+    /**
+     * @notice          Changes the endianness of a uint64.
+     * @dev             https://graphics.stanford.edu/~seander/bithacks.html#ReverseParallel
+     * @param _b        The unsigned integer to reverse
+     * @return          v - The reversed value
+     */
+    function reverseUint64(uint64 _b) internal pure returns (uint64) {
```

Comment:
couldn't it be handled on JS side?

Reply:
too complex, because I need both span (be) and getSpanValue (le)